### ISSUE TITLE
Jenkins Support

### ISSUE THREAD

**Issue by conwilly88** [coveralls-net/coveralls.net#42]
Just wondering if this supports Jenkins? If so is there any documentation around it?

Thanks

**Comment by jdeering** [comment]
Yes, Jenkins is supported and will be automatically detected when running in a Jenkins environment, setting build and git information appropriately.

We are currently lacking in the documentation for how to configuration Jenkins. I personally haven't used Jenkins, so I couldn't tell you off the top of my head. If you're familiar enough with Jenkins, the process should just be to add an "after build" script to run the coverage reports and send them to coveralls.io. Below are two Windows command-line options, depending on which testing framework you have set up.

```
# NUnit + OpenCover
> packages\OpenCover.4.6.166\tools\OpenCover.Console.exe -register:user -filter:"+[*]*" -target:"packages\NUnit.Runners.2.6.4\tools\nunit-console.exe" -targetargs:"/noshadow /domain:single YOURPROJECT.Tests\bin\debug\YOURPROJECT.Tests.dll" -output:coverage.xml

# MSTest + OpenCover
> packages\OpenCover.4.6.166\tools\OpenCover.Console.exe -register:user -filter:"+[*]*" -target:"C:\Program Files (x86)\Microsoft Visual Studio 12.0\Common7\IDE\MSTest.exe" -targetargs:"/testcontainer:YOURPROJECT.Tests/bin/debug/YOURPROJECT.Tests.dll" -output:coverage.xml

> packages\coveralls.io.1.3.4\tools\coveralls.net.exe --opencover coverage.xml -r %COVERALLS_REPO_TOKEN%
```

You would also need to do the first couple of steps in the [setup guide](https://github.com/coveralls-net/coveralls.net/wiki/CI-Integrations) for adding the package and getting your token(s) from coveralls.io.
